router/overlay_switch.go

```diff
@@ -50,6 +52,15 @@ func (osw *OverlaySwitch) Diagnostics() interface{} {
 	return diagnostics
 }
 
+func (d OverlaySwitchDiagnostics) Stats() *FastDPMetrics {
```

Comment:
Ghmm, I'd prefer to keep `OverlaySwitch` as generic as possible, i.e. to not include any details about a particular overlay implementation.